src/scancode/cli.py

```diff
@@ -655,7 +655,7 @@ def save_results(files_count, scanned_files, format, input, output_file):
         except HtmlAppAssetCopyError:
             echo_stderr('\nFailed to create HTML app.', fg='red')
 
-    elif format == 'json':
+    elif format == 'json' or format == 'json-pp':
```

Comment:
Do you mind adding some unit tests? Also some release tests in the `./etc/release/release.sh` script?